### PR TITLE
FTheoryTools: Use direct sum of Lie algebras

### PR DESCRIPTION
**Pull request by HereAround**
cc @emikelsons @apturner @lgoettgens

### CODE REVIEW

File: experimental/FTheoryTools/src/AbstractFTheoryModels/methods.jl [hunk 1]
```diff
@@ -653,8 +653,12 @@ function set_gauge_algebra(m::AbstractFTheoryModel, algebras::Vector{String})
       #return lie_algebra(C,Symbol('G'),parse(Int, g[3:end-1]));
     end
   end
-  gauge_algebras = [_construct(g) for g in algebras]
-  set_attribute!(m, :gauge_algebra => gauge_algebras)
+  if length(algebras) == 0
+    # Trivial Lie algebra not yet implemented?
```
Comment (lgoettgens) [inline experimental/FTheoryTools/src/AbstractFTheoryModels/methods.jl]: you could use `abelian_lie_algebra(::Field, 0)`

Comment (lgoettgens) [reply experimental/FTheoryTools/src/AbstractFTheoryModels/methods.jl]: or alternatively `direct_sum(::Field, LieAlgebra[])`

**Comment by lgoettgens** [comment]
I pushed my suggestions directly to this branch, as it seemed too complicated to write in a comment

**Comment by HereAround** [comment]
Great. Thank you @lgoettgens !

**Review by emikelsons — APPROVED** [review]
Looks good and I tested some stuff locally

**Pull request merged**